client/runtime/src/main/java/io/quarkiverse/openapi/generator/providers/AbstractAuthProvider.java

```diff
@@ -88,4 +99,15 @@ public final String getCanonicalAuthConfigPropertyName(String authPropertyName)
     public static String getCanonicalAuthConfigPropertyName(String authPropertyName, String openApiSpecId, String authName) {
         return String.format(CANONICAL_AUTH_CONFIG_PROPERTY_NAME, openApiSpecId, authName, authPropertyName);
     }
+
+    public static boolean isTokenPropagation(String openApiSpecId, String authName) {
```

Comment:
Same here regarding the `authName`. These methods now seems loosely coupled to this class, meaning that they might not be here.

Reply:
same reason as above: those are instance field, not class field/variable so I cannot use them in a static context

Reply:
Oh misread the static